tests/unit/controlgroup/controlgroup_methods.js

```diff
@@ -0,0 +1,120 @@
+module( "Controlgroup: methods" );
+
+test( "destroy", function() {
+	expect( 1 );
+	domEqual( ".controlgroup", function() {
+		$( ".controlgroup" ).controlgroup().controlgroup( "destroy" );
+	});
+});
+test( "disable", function( assert ) {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup().controlgroup( "disable" );
+	assert.lacksClasses( element, "ui-state-disabled",
+		"ui-state-disabled is not present on widget after disabling" );
```

Comment:
This message would be more helpful if it explained why the widget itself doesn't get the disabled class, instead of just describing the assertion.